test/suites/dev/test-precompile/test-precompile-conviction-voting.ts

```diff
@@ -180,5 +199,167 @@ describeSuite({
         expect(revertReason).to.contain("Must be an integer between 0 and 6 included");
       },
     });
+
+    it({
+      id: "T06",
+      title: "should allow to vote split",
+      test: async function () {
+        const ayes = 1n * 10n ** 18n;
+        const nays = 2n * 10n ** 18n;
+        // Vote split
+        const rawTx = await createViemTransaction(context, {
+          to: PRECOMPILE_CONVICTION_VOTING_ADDRESS,
```

Comment:
this can be refactored too

Reply:
I attempted to replace all the calls but after the test suit started to fail, I reverted the changes. I will give it another try.

Reply:
done